stores/governance/src/main/java/com/bloxbean/cardano/yaci/store/governance/processor/LocalGovActionStateProcessor.java

```diff
@@ -0,0 +1,60 @@
+package com.bloxbean.cardano.yaci.store.governance.processor;
+
+import com.bloxbean.cardano.yaci.store.events.EpochChangeEvent;
+import com.bloxbean.cardano.yaci.store.events.RollbackEvent;
+import com.bloxbean.cardano.yaci.store.governance.service.LocalGovStateService;
+import com.bloxbean.cardano.yaci.store.governance.storage.impl.repository.*;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnBean;
+import org.springframework.context.event.EventListener;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.concurrent.TimeUnit;
+
+@Component
+@RequiredArgsConstructor
+@Slf4j
+@ConditionalOnBean(LocalGovStateService.class)
+public class LocalGovActionStateProcessor {
```

Comment:
Should we introduce storage layer for LocalxxxRepository ? Otherwise, in case of another storage impl for goveranance, we still need a SQL DB only for this part.

Reply:
We will introduce storage layer for LocalxxxRepository